text_formatter.go

```diff
@@ -154,11 +154,12 @@ func (f *TextFormatter) needsQuoting(text string) bool {
 }
 
 func (f *TextFormatter) appendKeyValue(b *bytes.Buffer, key string, value interface{}) {
-
+	if b.String() != "" {
+		b.WriteByte(' ')
```

Comment:
I wonder whether the buffer could be a string slice (then we could just use `strings.Join()`). However this refactoring is out of scope of this PR an your change definitely does the job!